src/metadata/GithubActionMetadata.js

```diff
@@ -1,21 +1,92 @@
-import { DefaultMetadata } from 'leto-modelizer-plugin-core';
+import {
+  ComponentAttributeDefinition, ComponentDefinition,
+  DefaultMetadata,
+} from 'leto-modelizer-plugin-core';
+import metadata from 'src/assets/metadata/index';
+import GenericComponentDefinition from 'src/models/GenericComponentDefinition';
+import TriggerDefinition from 'src/models/TriggerDefinition';
 
 /**
  * Class to validate and retrieve components definitions from GithubAction metadata.
```

Comment:
is there multiple definition for one components ?

Reply:
yes